install.php

```diff
@@ -99,11 +113,37 @@ public function preflight($type, $parent)
 		$this->installRedcore($type, $parent);
 		$this->loadRedcoreLibrary();
 		$this->loadRedcoreLanguage();
-		$manifest  = $this->getManifest($parent);
-		$extensionType      = $manifest->attributes()->type;
+		$manifest = $this->getManifest($parent);
+		$extensionType = $manifest->attributes()->type;
+		$this->extensionName = strtolower($manifest->name);
```

Comment:
The name of the manifest does not have to be extension name, you should get element name instead

![localhost 127 0 0 1 red33test ram9d_extensions phpmyadmin 4 0 4 1](https://cloud.githubusercontent.com/assets/3668669/7061114/54177ed2-de8d-11e4-9b1c-71baeac90aa2.png)